ansible/roles/common/tasks/setup-wifi.yml

```diff
@@ -1,9 +1,19 @@
 ---
-- name: Configure WPA 
-  ansible.builtin.template:
-    src: templates/wpa_supplicant.conf.j2
-    dest: /etc/wpa_supplicant/wpa_supplicant.conf
-    mode: "0400"
+- name: Debug passphrase variables
+  debug:
+    var: wifi_settings.wifi_networks
+
+- name: Configure WiFi network
+  nmcli:
+    conn_name: "{{ item.name }}"
+    state: present
+    ssid: "{{ item.ssid }}"
+    priority: "{{ item.priority }}"
```

Comment:
priority should be autoconnect-priority

Reply:
> priority should be autoconnect-priority

I think that `autoconnect-priority` is not supported by the module: https://docs.ansible.com/ansible/latest/collections/community/general/nmcli_module.html the closest thing is priority but it is a different thing: `This is only used with ‘bridge’ - sets STP priority.`

I'm removing the priority... it should connect to the latest used.

Do we want to keep the `dean-4g` or can we just deprecate it in favour of the connect? 🤔

Reply:
indeed the module has no support. 
We need it so might something like : 

nmcli connection modify "elimu-connect" connection.autoconnect-priority 110

Reply:
Added as shell 👍

Reply:
yes this work fine. 
Please finish this with merge.